frontend/components/connectors.js

```diff
@@ -0,0 +1,31 @@
+import { connect } from 'react-redux';
+import { withRoute } from '@shopgate/engage/core';
+import { getBaseProductId } from '@shopgate/pwa-common-commerce/product/selectors/product';
+import { hasProductRelationsFiltered } from '../selectors';
+
+/**
+ * @param {string} type .
+ * @returns {Object}
+ */
+const makeMapStateToProps = type => (state, { route: { state: { productId } } }) => {
```

Comment:
why take the productId from routeState? why not using withProduct/useProduct

Reply:
Sheet is located outside of product context. We have no access to selection. Taking it from route works